apl/tabular-operators/distinct-operator.mdx

```diff
@@ -1,33 +1,145 @@
 ---
-title: 'distinct'
-description: 'Learn how to produce a table with the distinct combination of the provided fields of your dataset.'
-tags:
-    ['axiom documentation', 'documentation', 'axiom', 'tabular operators', 'distinct']
+title: distinct
+description: 'This page explains how to use the distinct operator function in APL.'
 ---
 
-Produces a table with the distinct combination of the provided Fields of the input table.
+The `distinct` operator in APL (Axiom Processing Language) returns a unique set of values from a specified column or set of columns. This operator is useful when you need to filter out duplicate entries and focus only on distinct values, such as unique user IDs, event types, or error codes within your datasets. Use the `distinct` operator in scenarios where eliminating duplicates helps you gain clearer insights from your data, like when analyzing logs, monitoring system traces, or reviewing security incidents.
 
-## Syntax
+## For users of other query languages
+
+If you come from other query languages, this section explains how to adjust your existing queries to achieve the same results in APL.
+
+<AccordionGroup>
+<Accordion title="Splunk SPL users">
+
+In Splunk’s SPL, the `dedup` command is often used to retrieve distinct values. In APL, the equivalent is the `distinct` operator, which behaves similarly by returning unique values but without necessarily ordering them.
+
+<CodeGroup>
+```splunk Splunk example
+index=web_logs
+| dedup user_id
+```
+
+```kusto APL equivalent
+['sample-http-logs'] 
+| distinct id
+```
+</CodeGroup>
+
+</Accordion>
+<Accordion title="ANSI SQL users">
+
+In ANSI SQL, you use `SELECT DISTINCT` to return unique rows from a table. In APL, the `distinct` operator serves a similar function but is placed after the table reference rather than in the `SELECT` clause.
+
+<CodeGroup>
+```sql SQL example
+SELECT DISTINCT user_id FROM web_logs;
+```
+
+```kusto APL equivalent
+['sample-http-logs'] 
+| distinct id
+```
+</CodeGroup>
+
+</Accordion>
+</AccordionGroup>
+
+## Usage
+
+### Syntax
 
 ```kusto
-| distinct Field1, Field2
+| distinct ColumnName1 [, ColumnName2, ...]
```

Comment:
I’m noticing that language is changing from Field to Column in this PR. 

Is that intentional? Fields is the language we use when describing limits, so this feels inconsistent.

Reply:
Good point,  corrected